src/note/asset_base.rs

```diff
@@ -48,7 +47,7 @@ pub const ZSA_ASSET_DIGEST_PERSONALIZATION: &[u8; 16] = b"ZSA-Asset-Digest";
 ///    Defined in [ZIP-226: Transfer and Burn of Zcash Shielded Assets][assetdigest].
 ///
 ///    [assetdigest]: https://zips.z.cash/zip-0226.html#asset-identifiers
```

Comment:
/// [assetdigest]: https://zips.z.cash/zip-0227#specification-asset-identifier-asset-digest-and-asset-base